sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -65,6 +65,11 @@ def test_fibonacci():
     assert fibonacci(n).limit(n, S.Infinity) == S.Infinity
     assert lucas(n).limit(n, S.Infinity) == S.Infinity
 
+    assert fibonacci(n).rewrite(sqrt) == \
+        2**(-n)*sqrt(5)*((1 + sqrt(5))**n - (-sqrt(5) + 1)**n) / 5
```

Comment:
Perhaps a test of the type ``fibonacci(n).rewrite(sqrt).subs(n, 10) == fibonacci(10)`` with a suitable simplify() (there are several variants) would be more to the point.